packages/main/src/TimePickerTemplate.tsx

```diff
@@ -26,7 +26,7 @@ export default function TimePickerTemplate(this: TimePicker) {
 					onFocusIn={this._onfocusin}
 					onKeyDown={this._onkeydown}
 				>
-					{this.valueStateMessage.length > 0 && !this.open &&
+					{this.valueStateMessage.length > 0 && !this.open && !this._isMobileDevice &&
```

Comment:
This expression is now 3 conditions in the template. This is perfect for a dedicated getter with a good name like `shouldShowValueStateMessage.......Somewhere`